src/main/webapp/jsp/adminEmailLog.jsp

```diff
@@ -14,7 +14,7 @@
     
     <%-- this form is used to store parameters for ajaxloader only --%>
     <form id="ajaxLoaderDataForm">
-        <input type="hidden" name="offset" value="${data.offset}">
+        <input type="hidden" name="offset" value="">
```

Comment:
Yes, now `offset` is not a part of `AdminEmailLogPageData` anymore (its value will be assigned via javascript) so I removed it.